english/dream/dream.md

```diff
@@ -5,3 +5,5 @@ You feel that there will be no consequence in your dream and you attempt to mani
 Afterwards, you decide to:
 
 [scream](../scream/scream.md)
+
+[lucid dream](../lucid/lucid.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.